packages/pds/src/index.ts

```diff
@@ -242,14 +248,39 @@ export class PDS {
       algos,
     })
 
-    let server = createServer({
+    const xrpcOpts: XrpcServerOptions = {
       validateResponse: config.debugMode,
       payload: {
         jsonLimit: 100 * 1024, // 100kb
         textLimit: 100 * 1024, // 100kb
         blobLimit: 5 * 1024 * 1024, // 5mb
       },
-    })
+    }
+    if (config.rateLimitsEnabled) {
+      xrpcOpts['rateLimits'] = {
+        creator: (opts: RateLimiterOpts) =>
+          RateLimiter.memory({
+            bypassSecret: config.rateLimitBypassKey,
+            ...opts,
+          }),
+        global: [
+          {
+            name: 'global-ip',
+            durationMs: 5 * MINUTE,
+            points: 1000,
+          },
+        ],
+        shared: [
+          {
+            name: 'repo-write',
+            durationMs: DAY,
+            points: 5000,
+          },
+        ],
```

Comment:
Just to confirm my understanding: this will amount to allowing 5000 create/update ops per day, and 1000 requests every 5mins by IP.  I take it on the consumer pds these limits are likely to be configurable as part of the deployment, but not necessary needed at the moment.

Reply:
yup that's exactly right 👍